src/baldr/tilehierarchy.cc

```diff
@@ -60,6 +60,32 @@ GraphId TileHierarchy::GetGraphId(const midgard::PointLL& pointll, const uint8_t
   return id;
 }
 
+// Returns the grid Id within the tile. A tile is subdivided into a 32x32 grid.
+// The grid Id within the tile is used to sort nodes spatially.
+const int32_t kGridDivisions = 32;
```

Comment:
should make this a mjolnir config so we can try different granularities and see what happens

Reply:
added as a mjolnir config